manifests/cf-manifest/operations.d/300-upgrade-cflinuxfs2-for-cve-mitigation.yml

```diff
@@ -0,0 +1,9 @@
+- path: /releases/name=cflinuxfs2/url
+  type: replace
+  value: https://storage.googleapis.com/cf-deployment-compiled-releases/cflinuxfs2-1.216.0-ubuntu-trusty-3586.16-20180612-151850-131406428.tgz
+- path: /releases/name=cflinuxfs2/version
+  type: replace
+  value: 1.216.0
```

Comment:
Also, any reason not to go all the way up to 1.219? (http://bosh.io/releases/github.com/cloudfoundry/cflinuxfs2-release?all=1)

Reply:
I didn't want to use hours-old releases for a hopefully-uneventful CVE fix.